meos/src/point/tpoint_tile.c

```diff
@@ -1143,6 +1143,7 @@ tpoint_space_time_split(Temporal *temp, float xsize, float ysize, float zsize,
       times[i] = DatumGetTimestampTz(box.period.lower);
     result[i++] = atstbox;
   }
+  pfree(state);
```

Comment:
This is not needed, we only exit the loop after freeing the state.

Reply:
Did you check if this `pfree` is needed or not?
I believe we always free the state before breaking out of the loop, so this new `pfree` should not be needed.